lang/cs/Org.Apache.REEF.Network/Org.Apache.REEF.Network.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
if possible, please remove the redundant change here.

Reply:
No idea how to fix this.

Reply:
Most likely the difference is the Unicode 0xFEFF marker inserted (or removed?) by the VisualStudio. You can open the file in binary mode e.g. in `vim -b` and fix that if you want.

Reply:
How does it look now?

Reply:
Looks good now!